tests/vcr_python_wrapper.py

```diff
@@ -81,15 +84,23 @@ def host_body_matcher(r1, r2):
     return body_json_l2_matcher(r1, r2)
 
 
-def filter_apipie_checksum(response):
-    # headers should be case insensitive, but for some reason they weren't for me
-    response['headers'].pop('apipie-checksum', None)
-    response['headers'].pop('Apipie-Checksum', None)
+def filter_response(response):
+    for header in FILTER_RESPONSE_HEADERS:
+        # headers should be case insensitive, but for some reason they weren't for me
+        response['headers'].pop(header.lower(), None)
+        response['headers'].pop(header, None)
     return response
 
 
-def filter_request_uri(request):
+def filter_request(request):
+    # always use foreman.example.org as the name of the system
     request.uri = urlunparse(urlparse(request.uri)._replace(netloc="foreman.example.org"))
+
+    for header in FILTER_REQUEST_HEADERS:
```

Comment:
Hmm, technically those could go to `filter_headers` of VCR directly. That sounds cleaner.

Reply:
yeah, that's better.